benchmarks/pytorch-lightning/utils.py

```diff
@@ -0,0 +1,126 @@
+""" Utility functions / classes / stuffs """
+from __future__ import annotations
+
+import inspect
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Callable, NewType, TypeVar
+
+import pl_bolts.datamodules
+import torch
+from pl_bolts.datamodules import CIFAR10DataModule
+from pl_bolts.datamodules.vision_datamodule import VisionDataModule
+from simple_parsing import choice
+from simple_parsing.helpers.serialization.serializable import Serializable
+from torch import nn
+from torchvision import models
+
+C = NewType("C", int)
+H = NewType("H", int)
+W = NewType("W", int)
+ModuleType = TypeVar("ModuleType", bound=nn.Module)
+
+DEFAULT_DATA_DIR: Path = Path(os.environ["MILABENCH_DIR_DATA"])
+
+VISION_DATAMODULES: dict[str, type[VisionDataModule]] = {
+    name.replace("DataModule", ""): cls
+    for name, cls in vars(pl_bolts.datamodules).items()
+    if inspect.isclass(cls) and issubclass(cls, VisionDataModule)
+}
+
+
+# NOTE: This isn't
```

Comment:
Don't leave us hanging. What is this not?? :)

Reply:
Fixed in 0cef9ea